src/BulletDynamics/ConstraintSolver/btContactSolverInfo.h

```diff
@@ -120,6 +122,7 @@ struct btContactSolverInfoDoubleData
 	double m_splitImpulseTurnErp;
 	double m_linearSlop;
 	double m_warmstartingFactor;
+	double m_articulatedWarmstartingFactor;
```

Comment:
note, we need to re-generate some serialization data structures, will do it after the commit.